cartography/intel/aws/ec2.py

```diff
@@ -72,6 +84,35 @@ def get_ec2_vpcs(session, region):
     return client.describe_vpcs()
 
 
+def load_ec2_key_pairs(session, data, region, current_aws_account_id, aws_update_tag):
```

Comment:
Please add an integration test for this function.  You can use this example with [dynamodb](https://github.com/lyft/cartography/blob/master/tests/integration/cartography/intel/aws/test_dynamodb.py) and its sample data [here](https://github.com/lyft/cartography/blob/master/tests/data/aws/dynamodb.py).

Basically just ensure that an EC2KeyPair node is actually ingested to Neo4j and connected to an AWS account.  Let us know if you need help with this!  We'll try to test your code internally tomorrow or early next week.

Reply:
Added. I think I might also like to test for relationships between keypairs and instances.